### ISSUE TITLE
VPinMame 954 not initializing roms correctly

### ISSUE THREAD

**Issue by rickmcclanahan** [vpinball/pinmame#273]
Installed 954 today and tried Dragon (gottlieb 1978) and got a rom error.  Went to vpinmame setup and test rom, it said it was missing files in the rom.  

Gave up and went to Genie, same error.  

Rolled back to VPinmame a few revisions back, maybe 940's or 950 and all roms work again.

**Comment by volkenborn** [comment]
Welcome to the wonderful world of emulation. 😄

As I found out just today, we were using a bad "CF" spider chip dump all these years, one byte being $09 where it really is $08.
So now we're using the correct one, and the bug I introduced in the PPS4 CPU emulation to account for this very ROM issue is gone.
The next release will therefore make use of the correct dump for all System 1 based games.

Because of the Gottlieb copyrights management still being enforced, all I can do is to point you to
https://github.com/garzol/pps4Emul/blob/main/A1752EFA1753EE.bin
where there is a copy of the EF / EE combination which is not supported yet but should work if split in two parts and used as u5_cf.bin and u4_ce.bin respectively, giving CRC errors of course...
